CHANGELOG.md

```diff
@@ -7,18 +7,39 @@ accidentally triggering the load of a previous DB version.**
 ## Unreleased
 
 **Bugfixes**
-* #2974 Fix index creation for hypertables with dropped columns
 * #2989 Refactor and harden size and stats functions
+
+**Thanks**
+* @pedrokost and @RobAtticus for reporting an issue with size
+  functions on empty hypertables
+
+## 2.1.1 (2021-03-29)
+
+This maintenance release contains bugfixes since the 2.1.0 release. We
+deem it high priority for upgrading.
+
+The bug fixes in this release address issues with CREATE INDEX and 
+UPSERT for hypertables, custom jobs, and gapfill queries.
+
+This release marks TimescaleDB as trusted extension.
```

Comment:
this is PG13 only

Reply:
I am fixing it according https://github.com/timescale/timescaledb/pull/3068#issuecomment-809539658